src/components/Teams/Teams.jsx

```diff
@@ -0,0 +1,85 @@
+import { useEffect, useState } from "react";
+import TeamMember from "../TeamMember/TeamMember";
+
+const Teams = () => {
+  const [isLoadedTeamsData, setIsLoadedTeamsData] = useState(false);
+  const [teamsData, setTeamsData] = useState([]);
+  const [teamMembersData, setTeamMembersData] = useState([]);
+  const [teamsError, setTeamsError] = useState(null);
+
+  useEffect(() => {
+  
+    //TODO: Add abort controller when fetching from API
+
+    setIsLoadedTeamsData(false);
+    // Fetch data from the JSON file in the public directory
+    async function fetchData () {
+      try {
+        const [responseTeams, responseMembers] =  await Promise.all([fetch("/teams.json"), fetch("/members.json")]);
+        // Check if both responses are successful
+        if (!responseTeams.ok || !responseMembers.ok) {
+          setTeamsError('Failed to fetch data!');
+        };
+        const dataTeams = await responseTeams.json();
+        const dataMembers = await responseMembers.json();
+
+        setTeamsData(dataTeams);
+        setTeamMembersData(dataMembers);
+      } catch (error) {
+        setTeamsError(error);
+        console.error("Error fetching data:", error);
+      } finally {
+        setIsLoadedTeamsData(true);
+      }
+    }
+    fetchData();
+    
+    //TODO: Add cleanup function for abort controller when fetching from API
+    
+  }, []);
+
+  // If data not yet loaded
+  if (!isLoadedTeamsData) {
+    return <div>Loading...</div>;
```

Comment:
```suggestion
    return <div aria-live="polite" role="status">Loading teams...</div>;
```

Here is the reference for [ARIA Live regions](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions).

The TLDR:
- Live region will announce what is going on to screen readers, we use the `polite` value here as it will not interrupt what the screen reader is reading. As the message about loading data is not urgent it is better to wait and not interrupt things.
- The `role="status"` identifies this `<div>` as a place where the status of the page is present.
- The added text helps give context of that is happening.